usecases/config/environment.go

```diff
@@ -195,6 +195,14 @@ func FromEnv(config *Config) error {
 		config.Persistence.LSMMaxSegmentSize = DefaultPersistenceLSMMaxSegmentSize
 	}
 
+	if err := parsePositiveInt(
+		"PERSISTENCE_LSM_SEGMENTS_CLEANUP_INTERVAL_HOURS",
```

Comment:
I think it should be `*_SECONDS` not `*_HOURS` yes?

Reply:
User facing setting is measured in hours for cleanup not to be executed to often. It is internally converted to seconds.